.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: 💬 Questions & Discussions
```

Comment:
@brlipi why is `Questions & Discussions` link being removed? 🤔 
We should keep it.

Reply:
I've replaced it now with two links:

- One for the "help" category
- One for the "general" category

What do you think about it?

Reply:
I think that brings too many options for the user that just clicked `New issue`.
Also, when accessing our `Discussions` and click `New discussion` those options will be asked again, so I don't see any advantage on adding more steps to the user to start a discussion.

My vote is to keep two links:
1. Sending users to https://github.com/devopness/devopness/discussions/categories/ideas-feature-requests
    - Possibly named "💡 Ideas & Feature Requests"
2. Sending users to https://github.com/orgs/devopness/discussions
  - `💬 Questions & Discussions`: feel free to propose new icon and title, but it should send users to the home of our `Discussions` tab

Reply:
I've reverted it to the `💬 Questions & Discussions` with the link to the repo's discussions page.